RealmSwift/List.swift

```diff
@@ -524,10 +524,3 @@ extension List: AssistedObjectiveCBridgeable {
         return (objectiveCValue: _rlmArray, metadata: nil)
     }
 }
-
-// MARK: Unavailable
-
-extension List {
```

Comment:
Because the migration assistance this provides is already provided by the more generic ones on `RealmCollection`, so it's redundant to add it here as well.